ntp-proto/src/algorithm/standard/mod.rs

```diff
@@ -192,24 +196,29 @@ impl<C: NtpClock, PeerID: Hash + Eq + Copy + Debug> TimeSyncController<C, PeerID
         id: PeerID,
         measurement: crate::peer::Measurement,
         packet: crate::NtpPacket<'static>,
-    ) -> Option<(Vec<PeerID>, TimeSnapshot)> {
+    ) -> StateUpdate<PeerID> {
         let now = NtpInstant::now();
 
         // Ignore measurements within a second of the last reset
         if let Some(reset) = self.last_reset {
             if now.abs_diff(reset) < NtpDuration::ONE {
-                return None;
+                return StateUpdate::default();
             }
         }
 
         // Update peer's state and check if the clock needs recalculation
         if !self.run_peer_update(now, id, measurement, packet) {
-            return None;
+            return StateUpdate::default();
         }
 
         self.recalculate_clock(now)
     }
 
+    fn time_update(&mut self) -> StateUpdate<PeerID> {
+        // Not needed for standard algorithm
+        StateUpdate::default()
+    }
```

Comment:
why not make it `unreachable!()` then?

Reply:
There is no harm in it accidentally getting called, so crashing the entire program for that seems harsh.

Reply:
fair enough